crates/testing/tests/standalone_integration_test.rs

```diff
@@ -139,3 +140,53 @@ fn test_call_query_macro() {
         },
     );
 }
+
+async fn bench_call<'a>(module: &ModuleHandle, call: &str, count: u32) -> Duration {
+    let json = format!(r#"{{"call": {{"fn": "{call}", "args": [{count}]}}}}"#);
+
+    let now = Instant::now();
+
+    module.send(json).await.unwrap();
+
+    now.elapsed()
+}
+
+#[test]
+#[serial]
+#[allow(clippy::disallowed_macros)]
+fn test_calling_bench_db() {
```

Comment:
One of the reason of the large numbers is because this reduces the impact of noise, and makes bigger the evidence of perf improvement/regresions.

Micro-benchmarks are less useful for checking our db engine, where is not kilobytes, but mega/giga data sizes we should start checking for.

It is not what criterion is for, but that is the bench suite we have.

Reply:
That's totally fair. I think this test is why the test suite is crashing below, though.